harvester_e2e_tests/integrations/test_4_vm_backup_restore.py

```diff
@@ -480,8 +480,18 @@ def test_restore_replace_with_delete_vols(
         spec = api_client.backups.RestoreSpec.for_existing(delete_volumes=True)
         code, data = api_client.backups.restore(unique_vm_name, spec)
         assert 201 == code, f'Failed to restore backup with current VM replaced, {data}'
-        vm_getable, (code, data) = vm_checker.wait_getable(unique_vm_name)
-        assert vm_getable, (code, data)
+
+        endtime = datetime.now() + timedelta(seconds=wait_timeout)
+        while endtime > datetime.now():
+            code, data = api_client.vms.get(unique_vm_name)
+            if 200 == code and "Running" == data.get('status', {}).get('printableStatus'):
```

Comment:
I feel that we can implement `wait_status_running` as the wait_status_stopped` does.

Reply:
Refactored in commit 58f5535.